lib/generators/react_on_rails/templates/client/server.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-console, func-names, no-var */
+var bodyParser = require('body-parser');
+var webpack = require('webpack');
+var WebpackDevServer = require('webpack-dev-server');
+var jade = require('jade');
+var sleep = require('sleep');
+var config = require('./webpack.client.hot.config');
+
+var comments = [
```

Comment:
It might make sense to have an option, defaulting to true, to copy the sample JSX files so that the node server can start.

At the min, let's have a comment referring how to setup basic REST API handling, with a reference to: https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/server.js